### ISSUE TITLE
[FOCUS MANAGEMENT]: Focus outline SHOULD be consistent - In this section button

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8974]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The focus outline **should** appear around all sides of the focused content, and in the current state the left side is cut off for the "In this section" button.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a keyboard user, I want to know with certainty where the focused area is, and for it to be a consistent cue across the site.

## Environment

* Operating System: all
* Browser: all
* Server destination: https://staging.va.gov/pittsburgh-health-care/

**Viewports less than 768px wide**

## Steps to Recreate

1. Enter https://staging.va.gov/pittsburgh-health-care/ in browser
2. Resize browser to less than 768 pixels wide
3. Select the "In this section" link
4. Verify the focus outline is cut off on the left side

### Current code

```diff
.va-sidenav-wrapper .va-sidenav-default-trigger {
    padding: 5px 0 0 22px;
    background-color: transparent;
    font-weight: 700;
    text-align: left;
    margin-bottom: 5px;
    position: relative;
    cursor: pointer;
    width: calc(100% - 25px);
}
```

### Possible fix

Reduce the padding a bit, and introduce more margin on the left to ensure the outline is contained.
```diff
.va-sidenav-wrapper .va-sidenav-default-trigger {
+    padding: 5px 0;
    background-color: transparent;
    font-weight: 700;
    text-align: left;
    margin-bottom: 5px;
+    margin-left: 1rem;
    position: relative;
    cursor: pointer;
    width: calc(100% - 25px);
}
```

## WCAG or Vendor Guidance (optional)

* [W3C - WCAG 2.0 Consistent Identification](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-functionality.html)

## Screenshots or Trace Logs

![Screen Shot 2020-05-11 at 3 56 15 PM](https://user-images.githubusercontent.com/57469/81606424-120bc380-93a1-11ea-83d0-9b00ea2c06b6.png)

**Comment by mmiddaugh** [comment]
This issue was called out during staging review so should be addressed, even though it is coded defect-3

**Comment by mmiddaugh** [comment]
@jenstrickland this is ready for your review

**Comment by jenstrickland** [comment]
Validated. 🎉
